src/search1_conversion/convert_result.py

```diff
@@ -30,30 +63,33 @@ def search_objects(params: dict, results: dict, meta: dict):
         'search_time': results['search_time'],
         'objects': objects,
     }
-    _add_access_group_info(ret, results, meta, post_processing)
-    _add_objects_and_info(ret, results, meta, post_processing)
+    _add_access_group_info(ret, results, ctx, post_processing)
+    _add_objects_and_info(ret, results, ctx, post_processing)
+
     return ret
 
 
-def search_types(params, results, meta):
+def search_types(_params, results, _ctx):
```

Comment:
You use _params and _ctx here but not later.  Is there a reason?

Reply:
Do you mean the underscore prefix? It is a way of indicating that the _ prefixed parameters are not being used. It is a common pattern. Unused parameters are flagged with a warning in my editor, although our code quality checks to ban them.

I did think about removing them completely from the call to search_types(..), but there is also a value in the transformation functions following the same pattern, even if some parameters are not used.

Reply:
I noticed they were being used.  Why not just drop them all together?  Is this an "interface" that needs to be common across a few classes?

Reply:
That was the idea, it was a de-facto interface, and it was more expeditious to just make the linter happy than to trace down the usages and adjust them. But, I too don't like the extra, useless parameters, so I'm removing them :)